modules/ROOT/pages/basic-alerts.adoc

```diff
@@ -40,4 +40,107 @@ The following levels of severity are available for basic alerts:
 
 include::partial$include-alert-severity.adoc[leveloffset=+1]
 
-// add basic alert email templates
+[[alert-variables]]
+== Basic Alert Email Template
+
+You can use the following variables in a basic alert email's *Subject* and *Message* fields.
+For the description of each variable, see <<var-defs>>.
+
+[%header,cols="25a,50a,20a"]
+.Variables for Alerts
+|===
+| Alert Type | Alert | Applicable Variables
+.6+| CloudHub App Alerts 
+|*CPU Utilization* .6+|* `${severity}`
+* `${operator}`
+* `${threshold}`
+* `${period}`
+* `${value}`
+* `${condition}`
+* `${organization}`
+* `${environment}`
+* `${alertLink}`
+* `${dashboardLink}`
+* `${resource}`
+| *Memory Utilization*
+| *Thread Count* 
+| *Message Count*
+| *Message Error Count*
+| *Message Response Time*
+
+.6+| Anypoint Runtime Fabric App Alerts 
+|*CPU Utilization* .6+|* `${severity}`
+* `${operator}`
+* `${threshold}`
+* `${period}`
+* `${value}`
+* `${condition}`
+* `${organization}`
+* `${environment}`
+* `${alertLink}`
+* `${dashboardLink}`
+* `${resource}`
+| *Memory Utilization*
+| *Thread Count* 
+| *Message Count*
+| *Message Error Count*
+| *Message Response Time*
+
+.3+|Hybrid App Alerts 
+| *Message Count* .3+| * `${severity}`
+* `${operator}`
+* `${threshold}`
+* `${period}`
+* `${value}`
+* `${condition}`
+* `${organization}`
+* `${environment}`
+* `${alertLink}`
+* `${dashboardLink}`
+* `${targetName}`
+* `${resource}`
+| *Message Error Count*
+| *Message Response Time*
+
+.3+|Hybrid Servers, Server Groups, and Cluster Alerts 
+| *CPU Utilization* .3+| * `${severity}`
+* `${operator}`
+* `${threshold}`
+* `${period}`
+* `${value}`
+* `${condition}`
+* `${organization}`
+* `${environment}`
+* `${alertLink}`
+* `${dashboardLink}`
+* `${memberName}`
```

Comment:
Here ${memberName} does not apply to hybrid servers. 
We might need to have a separate server section apart from serverGroup and clusters